web-ui/src/helpers/datetime.js

```diff
@@ -9,3 +9,13 @@ export const getQuarterBeginEnd = inputDate => ({
   startOfQuarter: startOfQuarter(inputDate),
   endOfQuarter: endOfQuarter(inputDate)
 });
+
+/**
+ * Return the current quarter number with year.
+ * @param {Date} date - The date to get the quarter number for.
+ * @returns {string} The quarter number with year.
+ */
+export const getQuarterDisplay = date => {
+  const quarter = Math.floor((date.getMonth() + 3) / 3);
```

Comment:
The Check-ins project already has a dependency on the date-fns library.
It provides several functions for working with quarters. Maybe those should be used.
See https://date-fns.org/v3.6.0/docs/getQuarter.

Reply:
Thanks, the `getQuarter` cleans the up the readability of this function.